src/irc/BridgedClient.ts

```diff
@@ -143,7 +143,7 @@ export class BridgedClient extends EventEmitter {
     }
 
     public get chanList() {
-        return this._chanList;
+        return Array.from(this._chanList);
```

Comment:
better to convert all calls to this to use `Set<string>`, but fine for now

Reply:
Yeah, maybe